### PR TITLE
update from lauras feedback

### PR DESCRIPTION
**Pull request by sethalt**
Reorganize readme
Add deployment section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,9 +10,9 @@ _Staging:_
 [![Trees-GitHub Issues](https://img.shields.io/github/issues/nciinc/fs-permit-platform.svg)](https://github.com/nciinc/fs-permit-platform/issues)
 [![Code Climate](https://codeclimate.com/github/nciinc/fs-permit-platform/badges/gpa.svg)](https://codeclimate.com/github/nciinc/fs-permit-platform)
 
-# U.S. Forest Service Intake Module
+# U.S. Forest Service Permit Platform
 
-** Module for intake of special use applications for Forest Service Application Permits. **
+** Platform for sale of Christmas tree permits and intake of special use applications for the Forest Service **
```
Comment (sadlerw) [inline README.md]: US Forest Service seems to be what they use on their web sites

File: README.md [hunk 2]
```diff
@@ -349,3 +351,15 @@ The file frontend/src/sass/\_focus-fix.scss implements a style fix in the upstre
 Due to performance issues, Pa11y testing is handled by an unreleased version of pa11y-ci 2.x that uses pa11y 5. Once pa11y-ci 2 is released, pa11y-ci dependency should be updated in `/frontend/package.json`.
```
Comment (sadlerw) [inline README.md]: this can be removed?

Comment (sethalt) [reply README.md]: good catch, yep i'll remove.

File: README.md (hunk 1 above)
Comment (sethalt) [reply README.md]: I'll update

**Pull request merged**